### DISCUSSION TITLE
Handle1 change via status effect causes a tool and a hand to move unrealistically fast.

### DISCUSSION THREAD

**Discussion by ThyGoodBoy** [FakeFishGames/Barotrauma#15448] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

The gun moves unrealistically fast whenever Handle1 is changed. This causes hand and gun movements to be snappy whenever Handle1 is changed via status effects, and the gun itself also changes its position despite the changes being made only to the Handle1.

What should have happened: upon the status effect triggering and changing the value of Handle1, the gun should stay in place, with the hand smoothly going to the new Handle1's position value.

The issue can be seen here.
![hmgload](https://github.com/user-attachments/assets/c8197543-e13b-4b75-a831-ad210000b198)
The back hand movement that is assigned to Handle2 is smooth, but whenever the Handle1 is changed, the front hand snaps to its position, while causing the gun to move as well, causing unnatural movement.

### Reproduction steps

1. Take any tool with Holdable component.
2. Give the tool a status effect that changes the Handle1 value.
3. Trigger the status effect.

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by jigardi** [comment]
@ThyGoodBoy 
First of all;  I tried your Guns Enhanced mod today  -  and it blew me away!  It's amazing! Thanks for makind such an awesome mod! : D

Second;  I very much hope this bug gets fixed,  so that you and anyone else following in your footsteps,  can take their mods to the next level! : D

Third;  do you know where the lines for this are in the source?  The odds of it being fixed will probably increase if we know why the bug happens.
